### ISSUE TITLE
Accessibility issues - low contrast, missing form labels, things out of order

### ISSUE THREAD

**Issue by podfeet** [bartificer/xkpasswd-js#49]
testing with WAVE Accessibility tool (web accessibility evaluation tool) reveals several problems.
